extensions/amp-powr-player/validator-amp-powr-player.protoascii

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-powr-player
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-powr-player"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-powr-player>
+  html_format: AMP
+  tag_name: "AMP-POWR-PLAYER"
+  requires_extension: "amp-powr-player"
+  attrs: { name: "autoplay" }
+  attrs: {
+    name: "data-account"
+    mandatory: true
+    value_regex: "[0-9a-zA-Z-]+"
+  }
+  attrs: {
+    name: "data-player"
+    mandatory: true
+    value_regex: "[0-9a-zA-Z-]+"
+  }
+  attrs: {
+    name: "data-video"
+    mandatory_oneof: "['data-video', 'data-terms']"
+    value_regex: "[0-9a-zA-Z-]+"
+  }
+  attrs: {
```

Comment:
nit: please list attributes alphabetically by name, IOW `data-terms` should go before `data-video`.